### PR TITLE
Improve PipelineRun documentation

### PR DESCRIPTION
**Pull request by tomfrenken**

### CODE REVIEW

**Comment by afrittoli** [comment]
Thank you for updating the docs.
Since this repo is not configured to squash commits on merge, cloud you squash them into one beforehand?

**Comment by tomfrenken** [comment]
Yeah, done

File: docs/pipelineruns.md [hunk 1]
```diff
@@ -220,9 +220,9 @@ allows to customize some Pod specific field per `Task` execution, aka
   [runtime class](https://kubernetes.io/docs/concepts/containers/runtime-class/)
   to use to run the pod.
 
-In the following example, the Task is defined with a `volumeMount`
-(`my-cache`), that is provided by the PipelineRun, using a
-PersistenceVolumeClaim. The Pod will also run as a non-root user.
+In the following example, the `Task` is defined with a `volumeMount`
+(`my-cache`), that is provided by the `PipelineRun`, using a
+`persistenceVolumeClaim`. The Pod will also run as a non-root user.
```
Comment (sbwsg) [inline docs/pipelineruns.md]: `persistentVolumeClaim`

**Pull request merged**